path_credentials.go

```diff
@@ -53,9 +53,9 @@ func (b *Backend) pathCredentialsRead(ctx context.Context, req *logical.Request,
 
 type walEntry struct {
 	UserName             string
-	ProjectID            string
-	OrganizationID       string
-	ProgrammaticAPIKeyID string
+	ProjectID            string `mapstructure:"project_id"`
```

Comment:
Is there a reason to use `mapstructure` in the decoding (i.e. in the rollback function)? We're JSON encoding this when it's written, so we should probably switch to JSON decoding.

Reply:
Because this same function is being used for the `WALRollback`, which receives an `interface{}` object.

Reply:
👍